pkg/asset/installconfig/aws/basedomain.go

```diff
@@ -28,8 +29,17 @@ func GetBaseDomain() (string, error) {
 		return "", err
 	}
 
+	// TODO Remove the hard code api endpoint after it's GA in AWS China
+	// If it's AWS China region, we have to set region to "cn-northwest-1"
+	// as route53 is hosted in that region.
+	awsConfig := aws.NewConfig()
+	switch *session.Config.Region {
+	case endpoints.CnNorth1RegionID, endpoints.CnNorthwest1RegionID:
+		awsConfig.WithRegion(endpoints.CnNorthwest1RegionID)
+		awsConfig.WithEndpoint("https://route53.amazonaws.com.cn")
+	}
 	logrus.Debugf("listing AWS hosted zones")
-	client := route53.New(session)
+	client := route53.New(session, awsConfig)
```

Comment:
> https://github.com/openshift/installer/pull/3230#issuecomment-594679340
> (1)

the terminal prompts should not support the aws china regions, only using the install-config.